internal/api/lib_test.go

```diff
@@ -298,12 +298,12 @@ func TestGetMetrics(t *testing.T) {
 	metrics, err = GetMetrics(cache)
 	assert.NoError(t, err)
 	require.Equal(t, uint32(1), metrics.HitsPinnedMemoryCache)
-	require.Equal(t, uint32(2), metrics.HitsMemoryCache)
-	require.Equal(t, uint32(1), metrics.HitsFsCache)
+	require.Equal(t, uint32(1), metrics.HitsMemoryCache)
+	require.Equal(t, uint32(2), metrics.HitsFsCache)
 	require.Equal(t, uint64(1), metrics.ElementsPinnedMemoryCache)
 	require.Equal(t, uint64(1), metrics.ElementsMemoryCache)
-	require.InEpsilon(t, 4075417, metrics.SizePinnedMemoryCache, 0.2)
-	require.InEpsilon(t, 4075417, metrics.SizeMemoryCache, 0.2)
+	require.InEpsilon(t, 2282344, metrics.SizePinnedMemoryCache, 0.2)
+	require.InEpsilon(t, 2282344, metrics.SizeMemoryCache, 0.2)
```

Comment:
This is pretty cool. Hope this is not a bug in the size estimation though and we really save that much memory.

Reply:
What's confusing me a bit here is that some of the tests fail on CI, but pass on my machine

Reply:
Yeah this is because the values are CPU and sometimes OS specific. You can use the values from the CI as expected and the 20% should make them work on your machine too 🤞

Reply:
Maybe you need to increase to 25% now:

```
                Error:          Relative error is too high: 0.2 (expected)
                                        < 0.24108197537268702 (actual)
```